### PR TITLE
docs: Article on Immutable Ledger in ERPNext

### PR DESCRIPTION
**Pull request by deepeshgarg007**

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!-- add-breadcrumbs -->
+# Immutable Ledger In ERPNext
+
+> Introduced in Version 13
+
+A major change has been introduced in ERPNext from version 13 onwards. This changes the way Accounting Ledger (General Ledger) and Stock Ledger works in ERPNext. There are multiple reasons why ledgers should be immutable. To list a few
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md:6]:
```suggestion
A major change has been introduced in ERPNext from version 13 onwards. This changes the way Accounting Ledger (General Ledger) and Stock Ledger works in ERPNext. There are multiple reasons why ledgers should be immutable. To list a few:
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+* Reposting future entries is computationally expensive.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md:8]:
```suggestion
* Reposting future entries is computationally expensive, i.e., slows down other transactions.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* In Stock Ledger, where the valuations are based on First-in-first-out(FIFO) method, the entire sequence may get regenerated which may upset valuations and profit for subsequent transactions.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md:9]:
```suggestion
* In Stock Ledger, where the valuations are based on First-in-first-out (FIFO) method, the entire sequence may get regenerated which may upset valuations and profit for subsequent transactions.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* Taxes paid for a period may also get changed
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md:10]:
```suggestion
* Taxes paid for a period may also get changed.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<!-- add-breadcrumbs -->
+# Immutable Ledger In ERPNext
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md:2]: add a note in Journal Entry, Stock Entry, Purchase Receipt, Delivery Note, wherever this applies. Something like

Note: From v13, rules for stock and accounting entries have changed with Immutable ledger. Backdated entries are no longer allowed and Journal Entries are cancelled by using a reverse entry. [Learn more here](/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext).

File: erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md (hunk 2 above)
Comment (deepeshgarg007) [reply erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md:8]: It doesn't slows down other transactions, the backdated transaction itself is computationally expensive as all future transactions needs to be updated

**Pull request merged**